.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "ra2",
+            "type": "clr",
+            "linux": {
+                "type": "mono"
+            },
+            "osx": {
+                "type": "mono"
+            },
+            "request": "launch",
+            "program": "${workspaceRoot}/engine/OpenRA.Game.exe",
+            "cwd": "${workspaceRoot}",
+            "args": ["Engine.LaunchPath=${workspaceRoot}", "Engine.ModSearchPaths=${workspaceRoot}/mods", "Game.Mod=ra2", "Debug.DisplayDeveloperSettings=true"]
```

Comment:
Shouldn't the `ModSearchPaths` include just `./mods` here as well? (See https://github.com/OpenRA/ra2/blob/master/launch-game.sh#L21)

Reply:
`${workspaceRoot}/mods` probably translates to just `./mods` already.

Reply:
`./mods` will be in the engine directory, won't it?

Reply:
I think that is relative to the repository root so it will be the `mods` folder where `ra2` is in.

Reply:
Then we'd be missing the mods folder of the engine directory or is that included in some other way?

Reply:
I can only say that is the exact same setup I use for @OpenHV and I never encountered problems during debugging.

Reply:
I'm just wondering since we include that in the start scripts (see link in the first comment). Perhaps the `modcontent` mod?

Reply:
Maybe. I never debugged during content installation.

Reply:
What is the status here. Does this work with the content mod or not?

Reply:
Amended it so it does work now.